decomposed/extensions/MDDE_LDM/Profile/BaseClassifierMapping/Templates/mdde_BaseClassifierMapping_XmlExport_Template.xml

```diff
@@ -16,6 +18,10 @@
   .foreach_item(mdde_Mapping_Examples,\n  &lt;Examples&gt;\n,\n  &lt;/Examples&gt;)
     %mdde_Example_XmlExport_Template%
   .next(\n)
+  .comment Add the content of the export extension template
+  .if (%Mapping_Template_Extension%)
```

Comment:
@willem-xbreeze Ik snap die opzet via .vbscript niet helemaal als ik het nu lees. Ik zie dat we dit zo in de 'oude' PDM extensie ook zo hebben opgezet, maar een template is toch altijd beschikbaar in GTL ongeacht in welke extensie deze zit?

Waarom doen we niet controleren of template bestaat met ? en dan inhoud verkrijgen?
Of moet je de template fully qualified aanroepen dan? %ExtensionName::ExportTemplateName% ?

Hieronder wat uitzoekwerk dump :).

Onderstaande werkt dus niet?
```
.if(%mdde_BaseClassifierMapping_Export_Template?%)
  %mdde_BaseClassifierMapping_Export_Template%
.endif
```

Even in de PowerDesigner handleiding gekeken en hier staat het volgende:
![image](https://github.com/CrossBreezeNL/PowerDesigner-MDDE-Extension/assets/97173058/d92f5236-9ef1-4db6-b09e-d1b076097e49)

Dit lijkt alleen voor extended attributes zo te zijn (niet getest).

In hoofdstuk 5.11.1 van Customizing en Extending staat stuk over templates aanroepen vanuit een extensie uit een language ext:
![image](https://github.com/CrossBreezeNL/PowerDesigner-MDDE-Extension/assets/97173058/fa7f49f5-bf86-4002-90a0-fea072ff81e0)

Ik vraag o.b.v. bovenstaande me af of het volgende kan werken:
```
.if(%MDDE_Model_Export_Extension::mdde_BaseClassifierMapping_Export_Template?%)
  %MDDE_Model_Export_Extension::mdde_BaseClassifierMapping_Export_Template%
.endif
```

Reply:
Verder hadden we in de 'oude' opzet xm_ als prefix in de model extension, en xme_ als prefix voor de export extensie zodat je kunt zien waar de template is. Nu gebruik je op beide plekken mdde_, misschien beter om voor de export extensie mddex_ te gebruiken en verder de naam gelijk te houden?

Dus voor mdde_BaseClassifierMapping_XmlExport_Template een export template mddex_BaseClassifierMapping_XmlExport_Template.

Reply:
Ga ik uitzoeken, ik had nu idd de bestaande werkwijze overgenomen uit PDM extensie

Reply:
@harmen-xb de opzet is met de laatste checkin aangepast zodat er geen extra VB script in de reguliere extensie zit. Bovengenoemde optie werkt helaas niet maar dit is dan second-best